util/openssl-update-copyright

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Copyright 2015-2018 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+#
+# openssl-update-copyright
+#
+# Updates the year ranges of all OpenSSL copyright statements to end with
+# the current year. If only a single year was given, that year is converted
+# to a range. All '(c)' and '(C)' signs are preserved.
+#
+# usage:
+#   openssl-update-copyright  <no-args>
+#
+
+utildir="$(dirname $(readlink -f $0))"
+ossldir="$(dirname $utildir)"
+
+this_year="$(date '+%Y')"
+some_year="[12][0-9][0-9][0-9]"
+
+for f in $(grep -rlE "Copyright ${some_year}.*The OpenSSL Project" "$ossldir"); do
```

Comment:
Could this be remade so the default is to get file names on the command line, but that `--all` will give all files?  I'm guessing a poor man's implementation could be something like this:

```
if [ "$1" = "--all" ]; then
    set -- $(grep -rlE "Copyright ${some_year}.*The OpenSSL Project" "$ossldir")
fi
for f in "$@"; do
```